src/lib.rs

```diff
@@ -36,6 +36,7 @@
 //! * [Simple layout/alignment functions - `embedded-layout`](https://crates.io/crates/embedded-layout)
 //! * [TextBox with text alignment options - `embedded-text`](https://crates.io/crates/embedded-text)
 //! * [Heapless plotting library for small embedded targets - `embedded-plots`](https://crates.io/crates/embedded-plots)
+//! * [Seven-segment displays - `eg-seven-segment`](https://crates.io/crates/eg-seven-segment)
```

Comment:
I admit I initially thought "e-g on seven segment displays? How?!" when I read this description. Should we maybe add some wording around "virtual" or "emulated" SS displays?

Reply:
That makes sense, I've added "virtual".